src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Base64Url/Base64UrlDecoder.cs

```diff
@@ -17,6 +19,10 @@ namespace System.Buffers.Text
     public static partial class Base64Url
     {
         private const int MaxStackallocThreshold = 256;
+#pragma warning disable CA1805 // Member 's_base64UrlByteDecoder' is explicitly initialized to its default value
+        private static Base64UrlDecoderByte s_base64UrlByteDecoder = default;
+        private static Base64UrlDecoderChar s_base64UrlCharDecoder = default;
```

Comment:
Having statics for these is a deoptimization. The call sites can just use eg `default(Base64UrlDecoderByte)`.